i18n/fr/docusaurus-plugin-content-docs/current/additional.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 3000
+---
+
+# 🛸 Ressources supplémentaires
+
+## Ressources
+
+Une vidéo d'introduction fantastique à PE : https://youtube.com/watch?v=dOxUroR57xs&feature=shares
+
+Un très bon guide d'ingénierie de prompts courts : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api
```

Comment:
per style guide in [the Notion that Brasé set up (see Discord)](https://discord.com/channels/1046228027434086460/1080591486741118996/1080597776196448327), we're keeping terms like "`Prompt Engineering`" in English (same with "`PE`", like on line 9) - the rationale is in the Notion --> Traduction --> Règles de traduction --> Traduction des textes. I guess it's for recognizability? and i suppose we should at least be consistent within the learnprompting.org course itself. Otherwise "`ingénierie de prompts`" could also be translated alternatively as "`ingénierie des invites`" or "`ingénierie d'invites`", which I've seen online. 

```suggestion
Un très bon guide court sur le Prompt Engineering : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api
```

(similar feedback for the 5 other instances of "`ingénierie de prompts`")

maybe [if we prompt ChatGPT to generate translations (see Discord chat)](https://discord.com/channels/1046228027434086460/1080591486741118996/1089918144748998787), it'd be good to include this:

```txt
Do not translate "Prompt Engineering" or "PE" into French, but keep it as "Prompt Engineering" and "PE", respectively. 
```

Reply:
> per style guide in [the Notion that Brasé set up (see Discord)](https://discord.com/channels/1046228027434086460/1080591486741118996/1080597776196448327), we're keeping terms like "`Prompt Engineering`" in English (same with "`PE`", like on line 9) - the rationale is in the Notion --> Traduction --> Règles de traduction --> Traduction des textes. I guess it's for recognizability? and i suppose we should at least be consistent within the learnprompting.org course itself. Otherwise "`ingénierie de prompts`" could also be translated alternatively as "`ingénierie des invites`" or "`ingénierie d'invites`", which I've seen online.
> 
> (similar feedback for the 5 other instances of "`ingénierie de prompts`")
> 
> maybe [if we prompt ChatGPT to generate translations (see Discord chat)](https://discord.com/channels/1046228027434086460/1080591486741118996/1089918144748998787), it'd be good to include this:
> 
> ```
> Do not translate "Prompt Engineering" or "PE" into French, but keep it as "Prompt Engineering" and "PE", respectively. 
> ```

@dr-nefardio3058 the other 5 instances of this in the file should also be fixed - i also added a suggestion for what to include in the prompt next time